src/nft/components/profile/list/NFTListingsGrid.tsx

```diff
@@ -184,6 +184,9 @@ const PriceTextInput = ({
           ref={inputRef}
           onChange={(v: FormEvent<HTMLInputElement>) => {
             const val = parseFloat(v.currentTarget.value)
```

Comment:
I would personally make it more explicit what edge case this is covering and do `if (v.currentTarget.value === '0.') return`

Reply:
done